### DISCUSSION TITLE
Supporting goose commands

### DISCUSSION THREAD

**Discussion by idan-gazit** [jackc/pgx#1292] [General]
Is there an option to somehow convert the Pool object into an sql.DB object? when running goose.up needs to pass sql.DB.

**Comment by jackc** [comment]
Sorry, no. You can go get a `*pgx.Conn` from a `*sql.DB`, but there is no way to go the other direction.

But you can use the exact same config to create a `*sql.DB` as you do to create a `*pgxpool.Pool`. If you are using conn strings you don't need to do anything special. Otherwise take a look at `RegisterConnConfig()`
